### PR TITLE
Add tickets key to control validation

### LINKED ISSUE #10868: Control feature for `tickets` appears broken
#### Description of problem:

When building products using control files with `tickets`, we get the following error:

```
RuntimeError: Unable to parse controls from /content/controls/cis_ocp_1_4_0.yml: Key tickets is not a valid for a control.
make[3]: *** [ocp4/CMakeFiles/ocp4-compile-all.dir/build.make:75: ocp4/ssg_build_compile_all-ocp4] Error 1
make[2]: *** [CMakeFiles/Makefile2:1954: ocp4/CMakeFiles/ocp4-compile-all.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:2353: ocp4/CMakeFiles/generate-ssg-ocp4-ds.xml.dir/rule] Error 2
make: *** [Makefile:1071: generate-ssg-ocp4-ds.xml] Error 2 
```

#### Steps to Reproduce:

1. Update a control file to use `tickets`
2. Run CI or attempt to build the profile

#### Actual Results:

```
RuntimeError: Unable to parse controls from /content/controls/cis_ocp_1_4_0.yml: Key tickets is not a valid for a control.
make[3]: *** [ocp4/CMakeFiles/ocp4-compile-all.dir/build.make:75: ocp4/ssg_build_compile_all-ocp4] Error 1
make[2]: *** [CMakeFiles/Makefile2:1954: ocp4/CMakeFiles/ocp4-compile-all.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:2353: ocp4/CMakeFiles/generate-ssg-ocp4-ds.xml.dir/rule] Error 2
make: *** [Makefile:1071: generate-ssg-ocp4-ds.xml] Error 2 
```

#### Expected Results:

The content should build without issues, as documented in https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#reporting-status.

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Add the ability to use `tickets` key in control files. 
#### Rationale:

Fixes #10868

### CODE REVIEW

File: ssg/controls.py [hunk 1]
```diff
@@ -118,8 +118,8 @@ def __hash__(self):
     def _check_keys(cls, control_dict):
         for key in control_dict.keys():
             if key not in cls.KEYS.keys() and key not in [
-                    'controls', 'original_title', 'related_rules', 'rules']:
-                raise ValueError("Key %s is not a valid for a control." % key)
+                'controls', 'original_title', 'related_rules', 'rules', 'tickets']:
```
[the remark this answers is not in the document]
Comment (Mab879) [reply ssg/controls.py:121]: I will work on follow up PR that merges these into `KEYS` once I have more information.

[the remark this answers is not in the document]
Comment (yuumasato) [reply ssg/controls.py]: As https://github.com/ComplianceAsCode/content/pull/10869 was merged we are not in a hurry.

[the remark this answers is not in the document]
Comment (yuumasato) [reply ssg/controls.py]: @Mab879 But we can merge this is if you don't plan to immediately tackle this.

[the remark this answers is not in the document]
Comment (Mab879) [reply ssg/controls.py]: I work on improving this PR today.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**